spec/mailers/notification_mailer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe NotificationMailer, type: :mailer do
+  describe 'Notification email' do
+    TEMPLATE_URL = '../../spec/fixtures/notification_mailer/test_email.html.slim'
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.